ui-components/src/page_components/listing/listing_sidebar/Apply.tsx

```diff
@@ -137,15 +137,21 @@ const Apply = (props: ApplyProps) => {
               </a>
             </p>
           ))}
-        {(listing.applicationPickUpAddress || listing.applicationPickUpAddressType) && (
+        {(listing.applicationPickUpAddress ||
+          listing.applicationPickUpAddressType ||
+          listing.buildingAddress) && (
           <>
             {!openDateState(listing) &&
               (onlineApplicationUrl !== "" || downloadMethods.length > 0) && (
                 <OrDivider bgColor="white" />
               )}
             <SubHeader text={t("listings.apply.pickUpAnApplication")} />
             <SidebarAddress
-              address={getAddress(listing.applicationPickUpAddressType, "pickUp")}
+              address={
+                getAddress(listing.applicationPickUpAddressType, "pickUp") ||
+                listing.applicationAddress ||
+                listing.buildingAddress
+              }
```

Comment:
I also wonder about whether we should tell users to pick up the application at the building if we don't know for sure that they can do so. @tomgebauer what do you think for this section? One alternative would be to just show the contact management company like you had in an earlier draft, but put it in this component.

Reply:
@kar-r-green as well

Reply:
My POV is that we show contact info for the management company and content to suggest they visit the management company's website (if the mgmt co has one) for application information or contact the management company directly by phone to request paper application.

@tomgebauer what do you recommend?

Reply:
@willrlin I think that @kar-r-green's solution is a great direction here. Just to be clear we're talking about something like this: 
![image](https://user-images.githubusercontent.com/3886741/130814996-7a6e4759-e63a-4f81-a746-23f0f5ac4293.png)
where we have widgets that can be rendered in the "right-rail" of the page depending on the data we have